lib/capybara/server/animation_disabler.rb

```diff
@@ -62,7 +62,7 @@ def directive_nonces(headers)
       end
 
       DISABLE_CSS_MARKUP_TEMPLATE = <<~CSS
-        %<selector>s, %<selector>s::before, %<selector>s::after {
+        %<selector>s, %<selector>s::before, %<selector>s::after, ::view-transition-group(*) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/::view-transition-group

My intention is that this would cover the root animation and any named animations.